FirebasePerformance/Sources/AppActivity/FPRAppActivityTracker.m

```diff
@@ -121,6 +150,68 @@ - (FIRTrace *)activeTrace {
   return self.backgroundSessionTrace;
 }
 
+/**
+ * Checks if prewarming is available for the platform on current device.
+ * It is available when running iOS 15 and above.
+ *
+ * @return true if the platform could prewarm apps on the current device
+ */
++ (BOOL)isPrewarmAvailable {
+  if (![[GULAppEnvironmentUtil applePlatform] isEqualToString:@"ios"]) {
+    return NO;
+  }
+  NSString *systemVersion = [GULAppEnvironmentUtil systemVersion];
+  if ([systemVersion length] == 0) {
+    return NO;
+  }
+  return [systemVersion compare:@"15" options:NSNumericSearch] != NSOrderedAscending;
```

Comment:
This check concerns me. Are there any other better way to do this? Doing a numeric search of "15" is super concerning. Eg: if we have a version that is 14.15.1, this check would return "YES" and it should not.